aiida_quantumespresso/calculations/__init__.py

```diff
@@ -264,12 +345,12 @@ def _generate_PWCPinputdata(cls, parameters, settings, pseudos, structure, kpoin
             if namelist in input_params:
                 # The following lines is meant to avoid putting in input the
                 # parameters like celldm(*)
-                stripped_inparams = [re.sub('[(0-9)]', '', _)
```

Comment:
Ah crap, that's because of the `# yapf: enable` above. Ignore everything in this file below this  - I'll put it back in its original state later.